isis/src/clipper/apps/jitterfit/jitterfit.xml

```diff
@@ -115,7 +167,7 @@
           This will allow the program to access the correct normal cube line number based on the 
           un-enlarged checktable values.
           Example: For a 4x enlargement set this value to 4.0.
-          ADD WHY
+          ...ADD WHY...
```

Comment:
I'm not sure if this is what the "add why" was, but was this just planning for an edge case, or do we expect the team to need to enlarge their cubes as part of their image processing process?

Reply:
One of the papers used to develop some of this had some info on this. I'll have to look that up and add more here